resources/views/newsletter/footer-subscription-form.blade.php

```diff
@@ -5,7 +5,6 @@
         name="subscribe_email"
         placeholder="Enter your email"
         model="email"
-        keydown-enter="subscribe"
```

Comment:
We listen for `keydown` on Enter, but since this is already wrapped in a form, enter will automatically submit the form.